File: Python/homepage.py
class Homepage(object):

    def __init__(self, driver):
        self.driver = driver

    def SignIn(self):
        self.driver.find_element_by_xpath('//a[text()="Log In"]').click()

    def SignUp(self):
        self.driver.find_element_by_xpath('//a[text()="Sign Up"]').click()
        self.driver.find_element_by_xpath('//button[text()="NEXT"]').click()

File: Java/src/PageFactory/HomePage.java
package PageFactory;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;
import org.openqa.selenium.support.PageFactory;

public class HomePage
{
    WebDriver driver;

    @FindBy(xpath="//h2[text()=\"How AutoGravity Works\"]")
    WebElement homePagetitle;

    public HomePage(WebDriver driver)
    {
        this.driver = driver;
        PageFactory.initElements(driver, this);
    }

    public String getTitle()
    {
        return homePagetitle.getText();
    }

    public void noSingIn()
    {
        driver.findElement(By.xpath("//button[text()=\"Get Started\"]")).click();
    }
}
